http/vibe/http/server.d

```diff
@@ -200,6 +201,16 @@ void handleHTTPConnection(TCPConnection connection, HTTPServerContext context)
 
 	scope (exit) connection.close();
 
+	// check wether the client's address is banned
+	foreach (ref virtual_host; context.m_virtualHosts)
+		foreach (ref ban_manager; virtual_host.settings.banManagers)
+		{
+			string remote_address = connection.remoteAddress().toAddressString();
+			ban_manager.onRequest(remote_address);
```

Comment:
calling onRequest will allow implementing more sophisticated BanManagers that could throttle the number of requests per clients